diff --git a/themes/doom-one-theme.el b/themes/doom-one-theme.el
index cdeb547..db8b09d 100644
--- a/themes/doom-one-theme.el
+++ b/themes/doom-one-theme.el
@@ -135,6 +135,7 @@ Can be an integer to determine the exact padding."
   (((line-number &override) :foreground base4)
    ((line-number-current-line &override) :foreground fg)
    ((font-lock-comment-face &override)
+    :slant 'italic
     :background (if doom-one-comment-bg (doom-lighten bg 0.05) 'unspecified))
    (mode-line
     :background modeline-bg :foreground modeline-fg
